packages/vue/public/assets/add_to_cart.svg

```diff
@@ -1,6 +1,6 @@
-<svg width="24" height="23" viewBox="0 0 24 23" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M10.3879 17.3167C8.97416 17.3167 7.80859 18.4823 7.80859 19.8959C7.80859 21.3096 8.97426 22.4752 10.3879 22.4752C11.8016 22.4762 12.9672 21.3105 12.9672 19.8967C12.9672 18.4828 11.8015 17.3164 10.3879 17.3164V17.3167ZM10.3879 20.9135C9.81715 20.9135 9.37045 20.4668 9.37045 19.896C9.37045 19.3253 9.81715 18.8786 10.3879 18.8786C10.9586 18.8786 11.4053 19.3253 11.4053 19.896C11.4043 20.4425 10.9334 20.9135 10.3879 20.9135Z" fill="white"/>
-<path d="M18.9211 17.3164C17.5074 17.3164 16.3418 18.4821 16.3418 19.8957C16.3418 21.3093 17.5075 22.475 18.9211 22.475C20.3347 22.475 21.5004 21.3093 21.5004 19.8957C21.4761 18.4829 20.3347 17.3164 18.9211 17.3164ZM18.9211 20.9132C18.3504 20.9132 17.9037 20.4665 17.9037 19.8958C17.9037 19.3251 18.3504 18.8784 18.9211 18.8784C19.4918 18.8784 19.9385 19.3251 19.9385 19.8958C19.9385 20.4423 19.4666 20.9132 18.9211 20.9132Z" fill="white"/>
-<path d="M15.0125 10.9126C14.9285 10.9126 14.8455 10.899 14.7651 10.8727C14.7571 10.8703 14.7509 10.8687 14.7446 10.8655C14.6535 10.8319 14.5741 10.7856 14.5071 10.7249L11.9364 8.53054C11.8007 8.41075 11.7266 8.24943 11.7266 8.07934C11.7266 7.90924 11.8007 7.74792 11.9364 7.62814C12.0703 7.50755 12.2507 7.44047 12.4409 7.44047C12.6311 7.44047 12.8114 7.50675 12.9454 7.62814L14.2982 8.73244V1.75509C14.2982 1.4029 14.6178 1.11621 15.0125 1.11621C15.4072 1.11621 15.7268 1.4021 15.7268 1.75509V8.73081L17.0796 7.6265C17.2144 7.50591 17.3939 7.43883 17.5841 7.43883C17.7743 7.43883 17.9546 7.50512 18.0886 7.6265C18.2234 7.74629 18.2984 7.90761 18.2984 8.0777C18.2984 8.2478 18.2243 8.40912 18.0886 8.5289L15.517 10.7233C15.45 10.7832 15.3714 10.8295 15.283 10.863C15.275 10.8654 15.267 10.8686 15.258 10.8718C15.1768 10.8998 15.0955 10.9126 15.0125 10.9126Z" fill="white"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M21.6229 3.947C22.3418 3.947 23.0366 4.29388 23.4833 4.86461L23.4832 4.86468C23.9551 5.43541 24.1034 6.17957 23.9299 6.8753L22.0695 14.5157C21.8214 15.5825 20.8534 16.3267 19.7624 16.3267H9.86547C8.77343 16.3267 7.83161 15.5825 7.55835 14.5157L4.53232 2.45966C4.4335 2.08758 4.11083 1.83952 3.73874 1.83952H1.20882C0.786334 1.83952 0.439453 1.49165 0.439453 1.07015C0.439453 0.647662 0.787326 0.300781 1.20882 0.300781H3.73874C4.83078 0.300781 5.7726 1.04495 6.04586 2.11179L6.51677 3.947H11.9507V5.5101H6.91403L9.07191 14.1667C9.17074 14.5388 9.49341 14.7869 9.86549 14.7869H19.7634C20.1355 14.7869 20.4582 14.5388 20.557 14.1667L22.4174 6.52638L22.4176 6.52656C22.4912 6.2785 22.4176 6.03044 22.2693 5.83277C22.1201 5.63413 21.872 5.5101 21.624 5.5101H18.0734V3.947H21.6229Z" fill="white"/>
+<svg width="25" height="23" viewBox="0 0 25 23" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
you should update it in shared/icons/icons.js :)

Reply:
yes, I updated this file too